src/browser.js

```diff
@@ -24,7 +24,7 @@ class RPC {
     this.fetch = fetch;
   }
 
-  request(rpcId: string, args: *): Promise<*> {
+  request(rpcId: string, args: *, headers: *): Promise<*> {
```

Comment:
Wondering if we could use a more specific type here for headers since it's always an object? Maybe:
```js
headers?: {[string]: string}
```